j2k/newSrc/org/jetbrains/kotlin/j2k/JavaToJKTreeBuilder.kt

```diff
@@ -24,21 +24,68 @@ import org.jetbrains.kotlin.j2k.tree.impl.*
 
 class JavaToJKTreeBuilder {
 
-    private class ExpressionTreeVisitor : JavaElementVisitor() {
-
-        private var resultExpression: JKExpression? = null
+    private object ExpressionTreeMapper {
+        fun PsiExpression.toJK(): JKExpression {
+            when (this) {
+                is PsiBinaryExpression -> {
+                    return toJK()
+                }
+                is PsiPrefixExpression -> {
+                    return toJK()
+                }
+                is PsiPostfixExpression -> {
+                    return toJK()
+                }
+                is PsiLiteralExpression -> {
+                    return toJK()
+                }
+                is PsiCallExpression -> {
+                    return toJK()
+                }
+            }
+            throw RuntimeException("Not supported")
```

Comment:
use `throw UnsupportedOperationException("Not supported")` if it's permament, or `TODO` if not